js/listeners.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+function Listeners() {
+  this.listeners = [];
+}
+
+Listeners.prototype.on = function(elm) {
+  var $elm = $(elm);
+  var args = _.toArray(arguments).slice(1);
+  this.listeners = this._listeners || [];
+  this.listeners.push({$elm: $elm, args: args});
+  $elm.on.apply($elm, args);
+};
+
+Listeners.prototype.off = function() {
```

Comment:
The purpose of this class is to keep track of event listeners so that they can be deleted in a batch, typically when the `destroy` method of a parent class gets called. We could allow this method to take optional arguments instead, but we haven't run into a use case for that yet. Or we could rename the method to something like `clear`.